index.html

```diff
@@ -105,27 +105,34 @@ <h1 class="brand-heading">Brain&nbsp;Imaging Data&nbsp;Structure</h1>
           <li><a href="https://github.com/INCF/BIDS2NDA">BIDS2NDA</a> (converter)</li>
-          <li><a href="https://github.com/nih-fmrif/BIDS-tools">AFNI BIDS-tools</a> (set of tools for converting to and analyzing BIDS datasets in AFNI)</li>
-          <li><a href="https://github.com/nipy/heudiconv">heudiconv</a> (converter)</li>
+          <li><a href="https://github.com/jmtyszka/bidskit">bidskit</a> (converter)</li>
+          <li><a href="https://github.com/dangom/dac2bids">dac2bids</a> (converter)</li>
           <li><a href="https://github.com/cbedetti/Dcm2Bids">Dcm2Bids</a> (converter)</li>
-		  <li><a href="https://github.com/jmtyszka/bidskit">bidskit</a> (converter)</li>
-		  <li><a href="https://github.com/dangom/dac2bids">dac2bids</a> (converter)</li>
-          <li><a href="http://fcp-indi.github.io/">C-PAC</a> (Configurable Pipeline for the Analysing Connectomes)</li>
+          <li><a href="https://github.com/neurolabusc/dcm2niix">DCM2NIIx</a> (converter)</li>
+          <li><a href="https://www.mathworks.com/matlabcentral/fileexchange/42997-dicom-to-nifti-converter--nifti-tool-and-viewer">DICM2NII</a> (converter)</li>
+          <li><a href="https://github.com/nipy/heudiconv">heudiconv</a> (converter)</li>
+          <li><a href="https://github.com/INCF/openfmri2bids">OpenfMRI2BIDS</a> (converter)</li>
+          </ul>
+          <li>Other Tools</li><ul>
+          <li><a href="https://github.com/rhodricusack/automaticanalysis">Automatic Analysis</a> (fMRI processing toolbox)</li>
           <li><a href="http://neuroimage.usc.edu/brainstorm/">Brainstorm</a> (MEG/EEG analysis package)</li>
-          <li><a href="http://bids-apps.neuroimaging.io">BIDS Apps</a> (a growing set of portable data processing pipelines that understand BIDS datasets)</li>
+          <li><a href="http://fcp-indi.github.io/">C-PAC</a> (Configurable Pipeline for the Analysing Connectomes)</li>
+          <li><a href="https://github.com/poldracklab/preprocessing-workflow">FMRIPREP</a> (preprocessing workflow)</li>
+          <li><a href="http://openfmri.org">OpenfMRI</a> (repository)</li>
+          </ul>
+          <li>Quality Assessment</li><ul>
+          <li><a href="http://mriqc.readthedocs.org/">MRIQC</a> (quality assesment)</li>
+          <li><a href="http://preprocessed-connectomes-project.org/quality-assessment-protocol/">QAP</a> (quality assesment)</li>
         </ul>
-        <p>Another good introduction to the BIDS standard can be found in the <a href="http://www.nature.com/articles/sdata201644">paper published in Nature Scientific Data</a>.</p>
+        <br>
+        <p>A good introduction to the BIDS standard can be found in the <a href="http://www.nature.com/articles/sdata201644">paper published in Nature Scientific Data</a>.<br>
+        A description of how to build BIDS containerized apps can be found in the <a href=" http://doi.org/10.1371/journal.pcbi.1005209">paper published in PLOS Computational Biology</a>.</p>
```

Comment:
Personally I feel that "how to build containerized apps supporting BIDS inputs" sounds better.